### ISSUE TITLE
[Mac] Unable to initialize db for geofences: Error: SQLITE_READONLY: attempt to write a readonly database

### ISSUE THREAD

**Issue by juju4** [prey/prey-node-client#261]
Reinstalled recently prey on a macos 10.11 laptop.
package install was fine but console can't update location or sent message.

in /var/log/prey.log
```
  PREY 1.6.5 spreads its wings!
  Current time: Wed Dec 28 2016 17:01:40 GMT+0100 (CET)
  Running with PID 84412 as prey over Node.js v4.4.5 on a x64 mac system (undefined)

     info Wed, 28 Dec 2016 16:01:40 GMT [actions] Starting action: geofencing
     info Wed, 28 Dec 2016 16:01:40 GMT [actions] Running: geofencing
     info Wed, 28 Dec 2016 16:01:40 GMT Plugin loaded: control-panel
     info Wed, 28 Dec 2016 16:01:40 GMT 1 plugins loaded with 0 errors.
     info Wed, 28 Dec 2016 16:01:40 GMT Initialized.
     info Wed, 28 Dec 2016 16:01:40 GMT [triggers] Watching: connection, network, power
     info Wed, 28 Dec 2016 16:01:40 GMT [actions] Starting trigger: connection
     info Wed, 28 Dec 2016 16:01:40 GMT [actions] Running: connection
     info Wed, 28 Dec 2016 16:01:40 GMT [actions] Starting trigger: network
     info Wed, 28 Dec 2016 16:01:40 GMT [actions] Starting trigger: power
     info Wed, 28 Dec 2016 16:01:41 GMT [actions] Running: network
     info Wed, 28 Dec 2016 16:01:41 GMT [actions] Running: power
    error Wed, 28 Dec 2016 16:01:41 GMT Unable to initialize db for geofences: Error: SQLITE_READONLY: attempt to write a readonly database
    error Wed, 28 Dec 2016 16:01:41 GMT Unable to initialize db for files: Error: SQLITE_READONLY: attempt to write a readonly database
    error Wed, 28 Dec 2016 16:01:41 GMT SQLITE_ERROR: no such table: files
 critical Wed, 28 Dec 2016 16:01:41 GMT UNCAUGHT EXCEPTION: logger.err is not a function
     info Wed, 28 Dec 2016 16:01:46 GMT Gracefully shutting down. Will self-destruct in 5 secs.
     info Wed, 28 Dec 2016 16:01:46 GMT [actions] Stopping: geofencing
     info Wed, 28 Dec 2016 16:01:46 GMT [actions] Stopping: connection
     info Wed, 28 Dec 2016 16:01:46 GMT [actions] Stopping: network
     info Wed, 28 Dec 2016 16:01:46 GMT [actions] Stopping: power
     info Wed, 28 Dec 2016 16:01:46 GMT [hooks] Unregistering hooks.
     info Wed, 28 Dec 2016 16:01:46 GMT [actions] Stopping: connection
     info Wed, 28 Dec 2016 16:01:46 GMT [actions] Stopping: network
     info Wed, 28 Dec 2016 16:01:46 GMT [actions] Stopping: power
     info Wed, 28 Dec 2016 16:01:46 GMT Shutdown complete. Have a jolly good day sir.
```

prey config check is fine but no prey-owned process is running
did another install on macos 10.12 and it's fine.

if I do as https://github.com/prey/prey-node-client/issues/192
```
/usr/local/lib/prey/current/bin/prey config hooks post_install
```
prey is now running and no initialization issue in prey.log.

have same logs than https://github.com/prey/prey-node-client/issues/180
```
Dec 28 17:14:47 myhost com.apple.xpc.launchd[1] (com.prey.agent): Service only ran for 8 seconds. Pushing respawn out by 2 seconds.
Dec 28 17:14:57 myhost com.apple.xpc.launchd[1] (com.prey.agent[95872]): Service exited with abnormal code: 1
Dec 28 17:14:57 myhost com.apple.xpc.launchd[1] (com.prey.agent): Service only ran for 8 seconds. Pushing respawn out by 2 seconds.
Dec 28 17:15:08 myhost com.apple.xpc.launchd[1] (com.prey.agent[95921]): Service exited with abnormal code: 1
Dec 28 17:15:08 myhost com.apple.xpc.launchd[1] (com.prey.agent): Service only ran for 8 seconds. Pushing respawn out by 2 seconds.
Dec 28 17:15:21 myhost com.apple.xpc.launchd[1] (com.prey.agent[95943]): Service exited with abnormal code: 1
Dec 28 17:15:38 myhost com.apple.xpc.launchd[1] (com.prey.agent[95976]): Service exited with abnormal code: 11
```

**Comment by javo** [comment]
Hi @juju4, could you try running this command in your terminal sudo chmod 644 /etc/prey/commands.db, then sudo pkill -f prx. Let me know how it goes :)

**Comment by juju4** [comment]
hello @javo 

as said, config hooks post_install fixed the problem
more improvement call, either an optional fix on 10.11 or maybe a warning at the end of install if service didn't start correctly.
